src/comm/bench-flow.js

```diff
@@ -331,9 +335,10 @@ module.exports.run = async function(configFile, networkFile) {
     //let configObject = require(absConfigFile);
     let configObject = Util.parseYaml(absConfigFile);
     let networkObject = require(absNetworkFile);
+    let defaultObject = Util.parseYaml(absDefaultConfigFile);
```

Comment:
Instead of manually accessing the config file, please use the built-in config mechanism of Caliper. Because this way you could only change these settings through the config file (and we want to avoid changing files, that's the purpose of the PR). You can access the configs this way:

```js
// at the beginning of the file
const config = require('./config.util.js');

// ...
// in the run function
let skipStart = config.getConfigSetting('core:skipStartScript', false);
// because command line options are parsed as strings
skipStart = skipStart === true || skipStart === 'true'; 
let skipEnd = config.getConfigSetting('core:skipEndScript', false);
skipEnd = skipEnd === true || skipEnd === 'true';

// now you can use them in the two if conditions
```